src/Message/CaptureResponse.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Omnipay\GoCardless\Message;
+
+use Omnipay\Common\Message\AbstractResponse;
+use Omnipay\Common\Message\RequestInterface;
+
+class CaptureResponse extends AbstractResponse
+{
+    protected $transactionReference;
+
+    public function __construct(RequestInterface $request, $data)
```

Comment:
This constructor isn't necessary. It will by default call the parent constructor.